x-pack/plugins/security/public/management/roles/edit_role/privileges/es/index_privileges.test.tsx

```diff
@@ -65,36 +65,84 @@ test('it renders a IndexPrivilegeForm for each privilege on the role', async ()
   const indicesAPIClient = indicesAPIClientMock.create();
   indicesAPIClient.getFields.mockResolvedValue(['foo']);
 
-  const props = {
-    role: {
-      name: '',
-      kibana: [],
-      elasticsearch: {
-        cluster: [],
-        indices: [
-          {
-            names: ['foo*'],
-            privileges: ['all'],
-            query: '*',
-            field_security: {
-              grant: ['some_field'],
-            },
+  const wrapper = mountWithIntl(
+    <KibanaContextProvider services={coreMock.createStart()}>
+      <IndexPrivileges
+        indexType="indices"
+        role={{
+          name: '',
+          kibana: [],
+          elasticsearch: {
+            cluster: [],
+            indices: [
+              {
+                names: ['foo*'],
+                privileges: ['all'],
+                query: '*',
+                field_security: {
+                  grant: ['some_field'],
+                },
+              },
+            ],
+            run_as: [],
           },
-        ],
-        run_as: [],
-      },
-    },
-    onChange: jest.fn(),
-    indexPatterns: [],
-    editable: true,
-    validator: new RoleValidator(),
-    availableIndexPrivileges: ['all', 'read', 'write', 'index'],
-    indicesAPIClient,
-    license,
-  };
+        }}
+        onChange={jest.fn()}
+        indexPatterns={[]}
+        editable
+        validator={new RoleValidator()}
+        availableIndexPrivileges={['all', 'read', 'write', 'index']}
+        indicesAPIClient={indicesAPIClient}
+        license={license}
+      />
+    </KibanaContextProvider>
+  );
+  await flushPromises();
+  expect(wrapper.find(IndexPrivilegeForm)).toHaveLength(1);
```

Comment:
Really minor nit: as we're testing that we the render a component for each index privilege, should we use a non-1 test case?  I realize this is how it has been (probably) forever, so feel free to ignore.

Reply:
Good idea, I'll update the tests